apps/two-dashboard-angular/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,3 @@
+export const environment = {
```

Comment:
Done?

Reply:
Nope. The environment.ts file has api_url and two_dashboard_api_url constants that are not in this file and the --prod build will fail because the references will be missing.